atomate2/openmm/flows/core.py

```diff
@@ -0,0 +1,51 @@
+
+from atomate2.openmm.jobs.core import InputMaker, EnergyMinimizationMaker, NPTMaker, AnnealMaker, NVTMaker
+from typing import List, Union, Optional
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from pydantic import Field
+from jobflow import Maker, Flow
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+
+
+
+@dataclass
+class ProductionMaker(Maker):
+    """
```

Comment:
To confirm my understanding, the ProductionMaker is used to make Flows for each simulation to be ran?

Reply:
Yes exactly, this is the full simulation run that one would use for a system of interest. The final NVT ensemble would produce data that one could use for a research paper.